### ISSUE TITLE
dotstar Featherwing

### ISSUE THREAD

**Issue by MBottinIUT** [adafruit/Adafruit_CircuitPython_DotStar#47]
Hi,
I have the Featherwing Dotstar module. I'm not an expert in CircuitPython, but the example on the 'https://learn.adafruit.com/dotstar-featherwing-in-circuitpython/text' page doesn't work.
In fact, I have a feeling that the 'dotstar_featherwing.py' module class doesn't support text scrolling anymore.
Am I wrong? If so, why didn't you leave these features in place?
Best regards,
Mike

**Comment by kevinjwalters** [comment]
This might belong in https://github.com/dastels/circuitPython_dotstar_featherwing ?

**Comment by dastels** [comment]
The adafruit_featherwing/dotstar_featherwing module is NOT the one used in the guide. The one used in the guide is linked via the community bundle and lives at [https://github.com/dastels/circuitPython_dotstar_featherwing](https://github.com/dastels/circuitPython_dotstar_featherwing). The DotStar constructor call was out of date (parameters were changed), and has been updated. I just tried the examples with CP 5.3.0 and they worked fine.
